### DISCUSSION TITLE
[Question] 关于 Microsoft Entra ID

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#4492] [提问] [answered]
Microsoft Entra ID 有没有要求必须订阅 Azure 的某项服务（包括免费和付费），或者可以无条件使用？

Microsoft Entra ID 注册应用程序后怎么添加账号和配置管理权限（管理页面拒绝我做任何高权限操作）

请求验证的 API 地址是什么？带什么参数？返回示例是什么？轮询验证结果的 API 地址又是什么？带的参数又是什么？验证成功的 API 响应示例是什么？如果取消操作，返回示例是什么？

**Comment by wuliaodexiaoluo** [comment]
A lot of question

**Comment by allMagicNB** [comment]
这和 PCL 有关吗？

**Reply by wuliaodexiaoluo** [reply]
![image](https://github.com/user-attachments/assets/cf90878c-350b-4504-b4fd-45bd66a44336)

~Microsoft Learn 那边暂时没人理我~

**Comment by Pigeon0v0** [comment, marked as answer]
参考 #3940
